examples/README.md

```diff
@@ -171,6 +172,31 @@ For this example the following is needed in the selected region:
 
 [hpc-cluster-high-io.yaml]: ./hpc-cluster-high-io.yaml
 
+### [slurm-gcp-v5-cluster.yaml] ![community-badge]
+
+This example creates an HPC cluster similar to the one created by
+[hpc-cluster-small.yaml], but uses modules built from version 5 of
+[slurm-gcp].
+
```

Comment:
Please add 1-2 sentences describing the debug and compute partitions and that you need to explicitly select the compute partition.